new/LIT7150Miracle.xml

```diff
@@ -0,0 +1,143 @@
+                <editor key="GS"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+
+                       
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                        This file is licensed under the Creative Commons
+                        Attribution-ShareAlike 4.0. 
+
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <abstract>
+                <p>
+                    Yǝsḥaq, a devoted monk under Bishop Gabriel of Alexandria, dedicated himself
+                    to fervent nightly prayers before the Virgin Mary's icon for seven years, bowing
+                    down one hundred times each night and imploring Jesus to reveal His mother to him.
+                    On the night of the feast of the 21st of Tāḥsās, the Virgin Mary appeared to him,
+                    acknowledging his prolonged suffer-ing and assuring him that God had heard his
+                    prayers. She promised that he would join her in three days, which came true as
+                    Yǝsḥaq peacefully passed away three days later and was re-ceived into her presence.
+                </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                    <term key="Religion"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="GS" when="2024-11-28">Created entity</change>
+            <change who="GS" when="2024-11-28">Added relations and description</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT7150Miracle" passive="LIT2384Taamme"/>
+                   <relation name="lawd:hasAttestation" active="LIT7150Miracle" passive="AkkMir6A"
```

Comment:
AkkMir6A is NOT an independent Id, it does not exist

it does not have an attestation (this is a relation for persons or places attested in texts), it is a PART of 5673, it is a VERSION of 3592, and it is DIFFERENT from 7151 - this must be expressed in appropriate relations